### PR TITLE
Allow reaction linking from force

### PR DESCRIPTION
**Pull request by zephraph**
This is just the other direction from #4475. It does the same thing (links reaction into force), but it allows you do to so _from_ force. 

So you'd run

```
yarn integrate reaction
```

and that'd start up the linking process. I need to add some docs still.

### CODE REVIEW

File: scripts/quicklink-to-project.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  exit 1
+fi
+
+cd $PROJECT
+
+# Does package have integrate command?
+cat package.json | jq -e .scripts.integrate &>2
+
+if [ $? -ne 0 ]; then
+  echo "$PROJECT doesn't have integrate npm script, ensure it's valid"
```
Comment (damassi) [inline scripts/quicklink-to-project.sh]: Maybe reference something that would make it valid?

Comment (zephraph) [reply scripts/quicklink-to-project.sh]: Like pointing to another project?

Comment (damassi) [reply scripts/quicklink-to-project.sh]: Or something -- a random dev won't know what "ensure it's valid' means.

Comment (alloy) [reply scripts/quicklink-to-project.sh]: And/or perhaps re-word it so people know to look at the `scripts` key of `package.json`? I’m not sure “npm scripts” is something everybody will immediately understand.

Comment (zephraph) [reply scripts/quicklink-to-project.sh]: I updated the wording a little bit. I'm gonna write a blog post and some more detailed documentation on this process this week so I'll follow up once that's done.

**Pull request merged**